morfologik-speller/src/main/java/morfologik/speller/Speller.java

```diff
@@ -400,14 +421,18 @@ private void addReplacement(List<CandidateData> candidates, String replacement)
    * @return A list of suggested candidate replacements.
    */
   public ArrayList<CandidateData> findReplacementCandidates(String word) {
+    return findReplacementCandidates(word, false);
+  }
+  
+  public ArrayList<CandidateData> findReplacementCandidates(String word, boolean evenIfWordInDictionary) {

```

Comment:
Can you make this two-argument method private? I don't think this should be exposed.